src/workflows/methods/SnapshotActionMenu.test.tsx

```diff
@@ -64,16 +73,30 @@ describe('snapshot action menu', () => {
     expect(deleteSnapshotButton).toBeInTheDocument();
     expect(deleteSnapshotButton).toHaveAttribute('aria-disabled', 'false');
     expect(screen.queryByRole('tooltip')).not.toBeInTheDocument();
+
+    // Act
+    const cloneSnapshotButton = screen.getByRole('button', { name: 'Clone snapshot' });
+    await user.pointer({ target: cloneSnapshotButton });
```

Comment:
Nit: The pointer movement action shouldn't be necessary in this or the next test if we are not checking for tooltips.